apps/ledger-live-desktop/tests/component/modal.component.ts

```diff
@@ -100,4 +100,13 @@ export class Modal extends Component {
       isVisible = await element.isVisible();
     }
   }
+
+  @step("Fill amount")
+  async fillAmount(amount: string) {
```

Comment:
this step also exist in [send.modal](https://github.com/LedgerHQ/ledger-live/blob/develop/apps/ledger-live-desktop/tests/page/modal/send.modal.ts#L80), you might consider making it global and remove one of them